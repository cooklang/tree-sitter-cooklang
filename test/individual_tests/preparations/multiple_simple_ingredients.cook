Add @salt and @pepper to taste.