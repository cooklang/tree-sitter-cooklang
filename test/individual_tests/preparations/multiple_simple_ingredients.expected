(recipe 
  (step 
    (ingredient (name))
    (ingredient (name))))